projects/spectator/test/test.ts

```diff
@@ -19,7 +19,9 @@ defineGlobalsInjections({
 });
 
 // First, initialize the Angular testing environment.
-getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting(), {
+    teardown: { destroyAfterEach: false }
+});
```

Comment:
This changes was made part of the Angular 13 migration scripts.
Is this change something that is useful for this library?

Reply:
Yes, the specs should run faster